### PR TITLE
add apps step

### LINKED ISSUE #7: Integration of "Apps" app into the setup process to provide suggestions of apps to install
During setup, there should be a screen where the user is shown a list of apps that they can install (such as sandboxed Google Play, Markup etc.) in order to make the onboarding process as smooth as possible.

At the moment, unclear if Play Services should be special cased and have its own step, or whether it should be generically included in this list of apps.

### PR DESCRIPTION
**Pull request by dot166**
lets you install and update apps during Setup Wizard

fixes #7 

needs https://github.com/GrapheneOS/Apps/pull/350

### CODE REVIEW

File: java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private lateinit var next: View
+
+    override fun bindViews() {
+        skip = findViewById(R.id.skip)
+        next = findViewById(R.id.next)
+    }
+
+    override fun setupActions() {
+        skip.setOnClickListener { SetupWizard.next(this) }
+        next.setOnClickListener { AppsActions.InstallApps(this) }
```
Comment (jiteshsingh) [inline java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt]: `AppsActions` doesn't have a method named `InstallApps(..)`

File: res/layout/activity_apps.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            android:layout_height="wrap_content"
+            android:text="@string/skip" />
+
+        <Space
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_weight="1" />
+
+        <Button
+            android:id="@+id/next"
```
Comment (jiteshsingh) [inline res/layout/activity_apps.xml]: button id should be `install`

File: java/app/grapheneos/setupwizard/action/AppsActions.kt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package app.grapheneos.setupwizard.action
+
+import android.app.Activity
+import android.content.Intent
+import android.util.Log
+import app.grapheneos.setupwizard.view.activity.SetupWizardActivity
+
+object AppsActions {
+    private const val TAG = "AppsActions"
+    private const val ACTION_APP_INSTALL = "app.grapheneos.setupwizard.action.APPS"
```
Comment (jiteshsingh) [inline java/app/grapheneos/setupwizard/action/AppsActions.kt:10]: where is this intent action being handled?

File: res/layout/activity_apps.xml (hunk 2 above)
Comment (dot166) [reply res/layout/activity_apps.xml]: done

File: java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt (hunk 1 above)
Comment (dot166) [reply java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt]: fixed

Comment (jiteshsingh) [reply java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt]: - better to follow java/kotlin method naming convention here? (first letter can't be caps)
- and `launchAppsInstaller` would be more appropriate name (following previous examples like `launchMigration`).

Comment (dot166) [reply java/app/grapheneos/setupwizard/view/activity/AppsActivity.kt]: done

**Comment by thestinger** [comment]
I think it makes sense to direct users to the fact that Apps exists by putting it on the default home screen and directing users to it. However, the apps available through it are not specifically recommended by GrapheneOS and I don't think putting this in the setup wizard is the right approach, particularly by reusing the same UI which is not meant to be used in this way. A starting point is improving the default home screen layout including putting Apps on it. There could be guides on using sandboxed Google Play and Android Auto within the OS by simply showing people this screen doesn't explain to them what those are and how to use them.

**Pull request closed without merging**